### DISCUSSION TITLE
Can't login with servers users

### DISCUSSION THREAD

**Discussion by erdebugger** [ltsp/ltsp#919] [General]
Hello.

I am using Debian 12 and I cannot login with the users who are registered on the server. 

I can login with all LDAP users and also with root because I have it in ltsp.conf, but it does not work with local users on the server.

If I use the ltsp.conf file in 

[server] 
OMIT_FUNCTIONS="remove_users" 

and in 

[clients]
OMIT_FUNCTIONS="pam_main"

I can enter the local chroot users but by default I cannot find a way to enter the users that are on the server locally.

Without any change in ltsp.conf in /etc/shadow they all appear, with the key as pamltsp but I cannot log in with any of them.

![image](https://github.com/ltsp/ltsp/assets/17746363/1b18cbf2-cf95-4ed0-be9d-3256cd7fd6f0)

I have sshfs installed and...

I can do ssh username@server too.

But then I can't login with this username in the ltsp client with Debían 12.

Please, help!

**Comment by franc0lb** [comment]
Hi, the same happened to me, and maybe I can help you, the problem was client's /etc/shadow, and my solution was to copy the server's /etc/shadow to the client, you can do it with post init commands.

For example: POST_INIT_MV_SHADOW="mv /etc/ltsp/shadow /etc/" 

But first of all you have to copy your /etc/shadow into /etc/ltsp/ or another directory shared with the clients, /etc/ltsp/ is automatically shared with the clients so you can use it.
After the client boots the command basically does mv of your /etc/shadow (previously copied to /etc/ltsp) to the client's /etc/ directory, you can replicate this command with any other files as well.
If you want to prevent the root account from being used on the clients, you can remove the first line of /etc/shadow (or any other account) in the copied file stored in /etc/ltsp/.
